### ISSUE TITLE
2021 [CVPR] (MRAA) Motion Representations for Articulated Animation

### ISSUE THREAD

**Issue by pomelyu** [pomelyu/paper-reading-notes#3]
## Introduction

[Github](https://github.com/snap-research/articulated-animation)
This paper is the enhancement from the FOMM[^1]. It provides an unsupervised(or self-supervised) method to identify and track object parts, and then articulate object for inference. In contrast to the keypoint-based method, the paper use region to describe the motion. The authors claim their method can animate a variety of objects, surpassing previous methods by a large margin on existing benchmarks.

## Method
<img width="1017" alt="image" src="https://user-images.githubusercontent.com/2679561/236134457-1ea208fb-19dd-443e-9e3a-80ae93267847.png">

- the region predictor predict the source(S) and target(T) K heatmaps
- calculate the affine transform from source to target through a whitened reference frame.
- BG motion predictor predictor the background transform
- pixel-wised flow predictor predict the optical flow and conference map by combining the K warped RGB image and gaussian-approximated heatmaps(4K) and the warped RGB background(3).
- the final autoencoder use the optical flow to warp the feature map and apply the confidence map for inpainting.
- Loss function: perceptual reconstruction loss and equivariance loss[^1]

## Highlight
- self-supervised training
- use **region** instead **learned key points**[^1] to articulate object. The results show the learned key points are always located in the edge of the corresponding part while region is almost aligned with the segmentation of the object.
- explicitly model the background/camera motion by an affine transform
- use additional disentangle network for shape and pose to animate the image, this method is different from the *standard* and *relative* method[^2] described in FOMM[^1].
- propose TED talk dataset

## Comments

[^1]: [FOMM] First order motion model for image animation
[^2]: suppose we have a source frame S and driving frames D1, D2, ..., and the reference frame D, which has the similar pose with S. The final results is calculate by applying D->D1, D->D2, ,,, to S
